host_cleanup.sh

```diff
@@ -28,3 +28,4 @@ sudo virsh net-destroy baremetal
 sudo virsh net-undefine baremetal
 sudo virsh net-destroy provisioning
 sudo virsh net-undefine provisioning
+sudo rm -rf /opt/dev-scripts/
```

Comment:
this should be $WORKING_DIR

Reply:
Also I'm not sure we should do this unconditionally, e.g in the typical dev workflow we do `make clean; make` or similar, and this will remove all the cached images thus increasing the test time and bandwidth usage.

Reply:
Ya, I agree with shardy here, this would break a lot of peoples workflow, as we would loose cached images,

Reply:
OK, perhaps it should just be noted in the readme to start clean this needs to be removed.